psignifit/sigmoids.py

```diff
@@ -3,16 +3,20 @@
 If you add a new sigmoid type, add it to the CLASS_BY_NAME constant
 and to the _LOGSPACE_NAMES, if it expects stimulus on an exponential scale.
 """
+from typing import Optional, TypeVar
+
 import numpy as np
-from typing import Optional
+import scipy
```

Comment:
by convention this is typically imported like
```python
import scipy as sp`
```
also, `scipy` is a lazy-import package, so given that below you are using `scipy.stats` you'll need to import it here explicitly:
```python
import scipy as sp
import sp.stats
```
The tests are passing probably because `scipy.stats` is imported somewhere else in the code, but not in this file.